templates/taxi/driver_list.html

```diff
@@ -7,7 +7,7 @@ <h1>Driver list</h1>
       {% for driver in driver_list%}
         <li>
           <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
```

Comment:
The URL tag uses double quotes, which is acceptable, but for consistency with other templates, consider using single quotes: `{% url 'taxi:driver-detail' pk=driver.id %}`.